scheduler/src/cook/mesos/api.clj

```diff
@@ -1015,39 +1015,72 @@
 ;;
 
 (def ReadRetriesRequest
-  {:job s/Uuid})
+  {(s/optional-key :job) s/Uuid
+   (s/optional-key :jobs) [s/Uuid]})
 (def UpdateRetriesRequest
-  (merge ReadRetriesRequest {:retries PosNum}))
+  (merge ReadRetriesRequest {(s/optional-key :retries) PosNum
+                             (s/optional-key :increment) PosNum}))
 
 (defn display-retries
   [conn ctx]
-  (:job/max-retries (d/entity (db conn) [:job/uuid (::job ctx)])))
+  (:job/max-retries (d/entity (db conn) [:job/uuid (-> ctx ::jobs first)])))
 
-(defn check-job-exists
+(defn jobs-from-request
+  [ctx]
+  (let [single-job (or (get-in ctx [:request :query-params :job])
```

Comment:
Ohhhhh @wyegelwel  I smell what you are cooking now.  Yes, I can change it to work like that.   Simpler.  Thanks